pkg/web/templates/partials/sunrise/sunrise_data.html

```diff
@@ -0,0 +1,51 @@
+<div class="flex items-center justify-between" hx-target="#sunrise-data" hx-swap="innerHTML">
+  <h2 class="text-xl font-semibold">Secure Message Data</h2>
+  <div>
+    <button type="button" class="max-w-fit btn bg-black text-white hover:bg-black/80">Download Message Data</button>
+  </div>
+</div>
+<div class="py-4">
+  
+  <div tabindex="0" class="my-4 collapse bg-base-200">
+    <div class="collapse-title text-xl font-semibold">Transaction Details:</div>
+    <div class="collapse-content">
+      <dl>
+        <div class="grid grid-cols-2">
+          <div class="transaction-preview-container">
+            <dt class="font-semibold">Amount</dt>
```

Comment:
This is a placeholder for now. It would be better to create a reusable component, but I haven't been able to get that to work inside partial files. During implementation, this will be updated along with the inclusion of error handling.